mesh/pmesh.hpp

```diff
@@ -143,12 +141,9 @@ class ParMesh : public Mesh
                     const std::unique_ptr<STable3D> &shared_faces,
                     int elem, int start, int end, const int fverts[][N]);
 
-   void GetFaceNbrElementTransformation(
-      int i, IsoparametricTransformation *ElTr);
-
    void GetGhostFaceTransformation(
-      FaceElementTransformations* FETr, Element::Type face_type,
-      Geometry::Type face_geom);
+      FaceElementTransformations &FElTr, Element::Type face_type,
+      Geometry::Type face_geom) const;
```

Comment:
This is changing interface -- users can use this method in derived classes. No need to change from pointer to reference at this point.

Reply:
Sure, this was mistakenly added as part of addressing [this comment]. Fixed in [39add0f](https://github.com/mfem/mfem/pull/4019/commits/39add0f7a9ef6eeca873fedd59ea75d31efac0f8).